src/readers/src/virtex_data_provider.cpp

```diff
@@ -0,0 +1,32 @@
+#include "virtex_data_provider.h"
+
+namespace {
+Box3D liftBox(Box3D box, float offset) {
```

Comment:
1) const
2) can we take box by ref?

Reply:
1 ok
2 BBox is just array of 3 float in this case, it is quite light-weighted structure, thus i decided leave it as is